dist/tools/cmake/generate-xcompile-toolchain.pl

```diff
@@ -1,26 +1,21 @@
 #!/usr/bin/env perl
```

Comment:
As moving to a general RIOT script, I would prefer to have it in either `sh/bash` or `python` than `perl`.

Reply:
yeah thought about that, too, but started with using what was already in place with `relic` - I think `sh` would be best.